packages/nextjs-kit/package.json

```diff
@@ -63,9 +63,9 @@
     "vitest": "^0.18.0"
   },
   "peerDependencies": {
-    "next": "12.1.6",
+    "next": "^12.2.5",
```

Comment:
I didn't run into this when setting this package up, but now I am realizing we should be less restrictive here, maybe `>=^12.2.5`. Currently when I try to `npm i` a clone of the drupal starter, it yells at me because the published `nextjs-kit` 1.0.0 wants exactly next 12.1.6.